dist/sgx/scripts/setup

```diff
@@ -72,121 +54,49 @@ function checkForPinFile() {
-        print_info "Enter the absolute path to the installation directory (empty directory name to abort)"
-        while true; do
-            read -p "> " INSTALL_DIR
-            if [[ "$INSTALL_DIR" == "" ]]; then
-                print_error "Aborted."
-                exit 1
-            fi
-            REAL_INSTALL_DIR=$(realpath /fs/$INSTALL_DIR)
-            INSTALL_DIR=/$(realpath $REAL_INSTALL_DIR --relative-to=/fs/)
-
-            if [ -d "$REAL_INSTALL_DIR" ]; then
-                print_error "Directory exists: $INSTALL_DIR. Try again or enter empty directory name to abort."
-            else
-                if ! mkdir "$REAL_INSTALL_DIR"; then
-                    print_error "Could not create directory: $INSTALL_DIR. Try again or enter empty directory name to abort."
-                else
-                    print_warning "powHSM will be installed to $INSTALL_DIR"
-                    break
-                fi
-            fi
-        done
-        print_info "Proceed? [Y/N]"
-        read -p "> " proceed
-        if [[ "Y" == "$proceed" ]] || [[ "y" == "$proceed" ]]; then
-            break;
-        else
-            rmdir $REAL_INSTALL_DIR
-        fi
-    done
-}
-
-function createServiceUnit() {
-    rm -rf $SERVICE_DIR
-    mkdir $SERVICE_DIR
-
-    cp $SCRIPTS_DIR/hsmsgx.service $SERVICE_DIR
-    # Replace the $HSM_INSTALL_DIR token in the script with the actual installation directory
-    sed -i "s|\$HSM_INSTALL_DIR|$INSTALL_DIR|g" $SERVICE_DIR/hsmsgx.service
-}
-
-function installPowHsm() {
-    mkdir $REAL_INSTALL_DIR/bin
-    cp -R $HSMBIN_DIR/* $REAL_INSTALL_DIR/bin
-    cp -R $ROOT_DIR/Dockerfile $REAL_INSTALL_DIR/bin
-}
-
-function startPowHsm() {
-    pushd $REAL_INSTALL_DIR > /dev/null
-    bin/$HSMSGX_BINARY bin/$HSMSGX_ENCLAVE_BINARY > /dev/null 2> /dev/null &
-    HSMSGX_PID=$!
-    popd > /dev/null
-}
-
-function stopPowHsm() {
-    kill $HSMSGX_PID
-}
-
 function createOutputDir() {
     rm -rf $EXPORT_DIR
     mkdir -p $EXPORT_DIR
 }
 
 function onboard() {
-    $ADMIN onboard
+    $ADMIN_CMD onboard
     error
 }
 
 function keys() {
-    $ADMIN pubkeys -uo $PUBLIC_KEY_FILE
+    $ADMIN_CMD pubkeys -uo $PUBLIC_KEY_FILE
     error
 }
 
 function attestation() {
-    $ADMIN attestation -o $ATTESTATION_FILE
+    $ADMIN_CMD attestation -o $ATTESTATION_FILE
     error
 }
 
 function verify_attestation() {
-    $ADMIN verify_attestation -t $ATTESTATION_FILE -b $PUBLIC_KEY_FILE_JSON
+    $ADMIN_CMD verify_attestation -t $ATTESTATION_FILE -b $PUBLIC_KEY_FILE_JSON
     error
 }
 
-
-print_info "Welcome to the SGX powHSM Setup for RSK"
+print_warning "Checking for required files..."
 checkForPinFile
-checkHsmBinaries
 expandBinaries
-selectInstallationDir
-createServiceUnit
-echo
-print_info "Installing the powHSM..."
-installPowHsm
-echo
-print_info "Starting the powHSM..."
-startPowHsm
-echo
 print_warning "Onboarding the powHSM..."
 onboard
-print_warning "Onboarding complete."
+print_info "Onboarding complete."
```

Comment:
Why did we change `print_warning` with `print_info` and viceversa? Was that part of the improvements?

Reply:
No strong reason, I just noticed the usage seemed a bit random as it was and tried to make it a bit more consistent:

warning / yellow -> I'm about to do something
info / green -> The current step succeeded
error / red -> The current step (or substep) failed

Reply:
I understand now. I say let's leave it as is and treat that as a separate improvement since we'd also have to take a look at/change the way it's done for the Ledger scripts.

Reply:
I thought we'd agreed on leaving the color scheme as it currently is on master?